### PR TITLE
Feature/compare

### PR DESCRIPTION
**Pull request by djarkimbaevbakyt6561**

### CODE REVIEW

File: src/api/compare.service.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { axiosInstance } from '../config/axiosInstants'
+
```
Comment (DanielKBV) [inline src/api/compare.service.js:2]: Cannot convert undefined or null to object
TypeError: Cannot convert undefined or null to object
authSlice.js:19

**Comment by Kamchybekk** [comment]
Screen Shot 2023-09-11 at 18 00 45

перейти в справнение

**Comment by Kamchybekk** [comment]
Screen Shot 2023-09-11 at 18 02 41

**Comment by Kamchybekk** [comment]
Screen Shot 2023-09-11 at 18 04 54

**Comment by Kamchybekk** [comment]
Screen Shot 2023-09-11 at 18 09 30

**Comment by Kamchybekk** [comment]
Screen Shot 2023-09-11 at 18 18 15

**Pull request merged**